app/models/user.rb

```diff
@@ -20,10 +18,6 @@ def admin?
     false
   end
 
-  def generate_slug
-    self.slug = display_name.parameterize unless display_name.nil?
-  end
-
```

Comment:
No longer need this since now our dispaly_name validates there is something there and it would work in a path